.github/workflows/grapl-release.yml

```diff
@@ -27,9 +26,17 @@ jobs:
 
           echo '::set-env name=VERSION::$VERSION'
 
+      - name: Get upload URL for release corresponding to this release event
+        run: |
+          URL = "$(curl --request GET \
```

Comment:
This mess seems like it could leak the github token...

Reply:
I don't think so. It's an HTTPS request to github. The only thing I could imagine is that we accidentally log something - are github action logs even public? Maybe run curl in quiet mode? '-o /dev/null' maybe

Reply:
Ah, I  guess if it logs the exec itself.

Reply:
Yeah what I'm worried about is something like
1. `curl` errors and leaks the header to stdout
2. `python` errors and leaks the header via a stacktrace
3. `$UPLOAD_URL` now has the `$GITHUB_TOKEN` embedded in it
4. `Upload sysmon-subgraph-generator to Github` fails and leaks the `$GITHUB_TOKEN` back to the UI.. or something..

Reply:
You could pipe to an intermediary file, then read that through Python. Hard to imagine that messing up, if Python errors it'll be a key error, or an IO error, it wouldn't print the whole thing.

Reply:
Yeah I played with it some, it seems safe.